boms/common-ee/pom.xml

```diff
@@ -2386,6 +2458,18 @@
                 </exclusions>
             </dependency>
 
+            <dependency>
+                <groupId>software.amazon.ion</groupId>
+                <artifactId>ion-java</artifactId>
+                <version>${version.com.amazon.ion}</version>
```

Comment:
@rhusar FYI:

https://amzn.github.io/ion-docs/news/2020/01/15/software.amazon.ion-deprecated.html

This is just FYI, as this dep comes via com.amazonaws:aws-java-sdk-core:jar:1.12.284, but even the latest release of that still uses software.amazon.ion:

https://repo1.maven.org/maven2/com/amazonaws/aws-java-sdk-core/1.12.334/aws-java-sdk-core-1.12.334.pom

Reply:
Yup... they are a bit messy. I will keep an eye on that when they clean that up.